### PR TITLE
Layout: updates story from knobs to control

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to control.
**Which issue(s) this PR fixes**:

Relates to #30182

### CODE REVIEW

File: packages/grafana-ui/src/components/Layout/Layout.story.tsx [hunk 1]
```diff
@@ -15,31 +15,59 @@ export default {
...
-    <HorizontalGroup justify={justify as any} spacing={spacing as any}>
+    <HorizontalGroup {...args}>
       <Button>Save</Button>
       <Button>Cancel</Button>
     </HorizontalGroup>
   );
 };
+Horizontal.args = {
+  orientation: Orientation.Horizontal,
+};
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Layout/Layout.story.tsx:61]: Sorry for being a complete Horizontal/VerticalGroup newb but is this prop really needed? It wasn't needed before?

File: packages/grafana-ui/src/components/Layout/Layout.story.tsx [hunk 2]
```diff
@@ -15,31 +15,59 @@ export default {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['orientation'],
+    },
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Layout/Layout.story.tsx:20]: Shouldn't we disable knobs too or are we disabling all knobs when everything is migrated?
```suggestion
    controls: {
      exclude: ['orientation'],
    },
    knobs: {
      disabled: true,
    },
```

**Review by hugohaggmark — COMMENTED** [review]
Looks good 🎉  I added some comments for you to consider.

File: packages/grafana-ui/src/components/Layout/Layout.story.tsx (hunk 1 above)
Comment (thisisobate) [reply packages/grafana-ui/src/components/Layout/Layout.story.tsx:61]: yeah...you're right. It's not needed at all

File: packages/grafana-ui/src/components/Layout/Layout.story.tsx (hunk 2 above)
Comment (thisisobate) [reply packages/grafana-ui/src/components/Layout/Layout.story.tsx:20]: good catch Hugo! Thanks!

**Review by hugohaggmark — APPROVED** [review]
LGTM 🎉

**Pull request merged**